src/App.tsx

```diff
@@ -1,61 +1,155 @@
+import { FormEvent, useState } from 'react';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
+function getUser(todoUserId: number): User | null {
+  const userObj = usersFromServer.find(user => user.id === todoUserId);
+
+  return userObj || null;
+}
+
+export const newTodos: Todo[] = todosFromServer.map((todo) => {
+  return {
+    id: todo.id,
+    title: todo.title,
+    userId: todo.userId,
+    completed: todo.completed,
+    user: getUser(todo.userId),
+  };
+});
 
 export const App = () => {
+  const [title, setTitle] = useState('');
+  const [userName, setOption] = useState('0');
+  const [tittleError, setTitleError] = useState(false);
+  const [userError, setUserError] = useState(false);
+  const [todos, setTodo] = useState(newTodos);
+
+  function newTodoId() {
+    let { id } = todos[0];
+
+    for (let i = 0; i < todos.length; i += 1) {
+      if (id < todos[i].id) {
+        id = todos[i].id;
+      }
+    }
+
+    return id;
+  }
+
+  const handlerSubmit = (event: FormEvent) => {
+    event.preventDefault();
+    if (title !== '' && userName !== '0') {
+      const user = usersFromServer
```

Comment:
you have function for find user, reuse it

Reply:
I don't have getUserByName function